### PR TITLE
shanoir-issue#2222: upgrade ohif viewer

### PR DESCRIPTION
**Pull request by pierrehenri-dauvergne**

### CODE REVIEW

File: docker-compose/nginx/viewer/app-config.js [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+            component: 'cornerstoneViewportComponent',
+          },
+        ];
+      },
+      getSopClassHandlerModule: function() {
+        return [
+          {
+            name: 'cornerstone',
+            sopClassUIDs: [
+              '1.2.840.10008.5.1.4.1.1.2', // CT Image Storage
```
Comment (michaelkain) [inline docker-compose/nginx/viewer/app-config.js:19]: Hi @pierrehenri-dauvergne, excellent, this PR looks very promising :-)! Please add more SOP Class UIDs here for other modalities supported by Shanoir: PT, NM.
'1.2.840.10008.5.1.4.1.1.2.1' , // Enhanced CT Image Storage
'1.2.840.10008.5.1.4.1.1.4.1', // Enhanced MR Image Storage
'1.2.840.10008.5.1.4.1.1.4.3', // Enhanced MR Color Image Storage
'1.2.840.10008.5.1.4.1.1.130', // Enhanced PET Image Storage
'1.2.840.10008.5.1.4.1.1.20', // Nuclear Medicine Image Storage

File: docker-compose/nginx/viewer/app-config.js [hunk 2]
```diff
@@ -1,153 +1,49 @@
...
+      namespace: '@ohif/extension-default.dataSourcesModule.dicomweb',
+      sourceName: 'dicomweb',
+      configuration: {
+        friendlyName: 'DCM4CHEE Server',
         name: 'SHANOIR-NG',
-        wadoUriRoot: 'SHANOIR_VIEWER_OHIF_URL_SCHEME://SHANOIR_VIEWER_OHIF_URL_HOST/shanoir-ng/',
-        qidoRoot: 'SHANOIR_VIEWER_OHIF_URL_SCHEME://SHANOIR_VIEWER_OHIF_URL_HOST/dicomweb',
-        wadoRoot: 'SHANOIR_VIEWER_OHIF_URL_SCHEME://SHANOIR_VIEWER_OHIF_URL_HOST/dicomweb',
-        qidoSupportsIncludeField: true,
+        wadoUriRoot: 'SHANOIR_VIEWER_OHIF_URL_SCHEME://SHANOIR_VIEWER_OHIF_URL_HOST/pacs',
```
Comment (michaelkain) [inline docker-compose/nginx/viewer/app-config.js:19]: Hi @pierrehenri-dauvergne, thank you for your PR. I think, the URIs should not be changed at all. Your PR does not change anything in the backend, but when you change the URIs how the viewer will get his information from shanoir?

File: docker-compose/nginx/viewer/app-config.js [hunk 3]
```diff
@@ -1,153 +1,49 @@
...
-      label: 'W/L Preset 9',
-      keys: ['9'],
-    },
+      authority: 'SHANOIR_URL_SCHEME://SHANOIR_URL_HOST/realms/shanoir-ng',
+      client_id: 'ohif-viewer',
+      redirect_uri: 'SHANOIR_VIEWER_OHIF_URL_SCHEME://SHANOIR_VIEWER_OHIF_URL_HOST/callback',
+      response_type: 'code',
+      scope: 'openid',
+      post_logout_redirect_uri: 'SHANOIR_URL_SCHEME://SHANOIR_URL_HOST',
+	  revoke_uri: 'https://accounts.google.com/o/oauth2/revoke?token=',
```
Comment (michaelkain) [inline docker-compose/nginx/viewer/app-config.js:43]: this can not be correct

File: docker-compose/Dockerfile [hunk 4]
```diff
@@ -311,7 +311,7 @@ CMD []
 
 ################ nginx #####################################################
 
-FROM ohif/viewer:v4.12.32.19362 AS nginx-viewer
+FROM ohif/app:v3.9.0-beta.63 AS nginx-viewer
```
Comment (michaelkain) [inline docker-compose/Dockerfile:314]: very good

**Comment by pierrehenri-dauvergne** [comment]
Another PR was made : #2317

**Pull request closed without merging**